pkg/network/render.go

```diff
@@ -338,13 +338,16 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	case len(prev.ServiceNetwork) > len(next.ServiceNetwork):
 		// Going from dual to single
 		dualStack, singleStack = prev, next
+	case !reflect.DeepEqual(prev.ServiceNetwork, next.ServiceNetwork):
+		return errors.Errorf("unsupported change to ServiceNetwork")
+	case validateCidrExpansion(prev.ClusterNetwork, next.ClusterNetwork) &&
```

Comment:
Can this be moved to `IsChangeSafe` instead? it looks like a better place to verify changes